CMakeLists.txt

```diff
@@ -27,18 +27,34 @@ project(Mir)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 
-set(MIR_VERSION_MAJOR 0)
-set(MIR_VERSION_MINOR 29)
-set(MIR_VERSION_PATCH 0)
+if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/mir-version)
```

Comment:
Think we should make this the other way? As in, if `git describe` completes, use it, even if mir-version's there?

Also, isn't `VERSION` the _canonical_ name of the file holding this?